firmware/tunerstudio/rusefi.input

```diff
@@ -1220,6 +1223,15 @@ menuDialog = main
 		subMenu = oilPressureSensor,		"Oil pressure"
 		subMenu = egtInputs,				"EGT" @@if_ts_show_egt
 
+menu = "&Electronic Throttle"
+
+		subMenu = etbDialog,				"Electronic Throttle Settings" @@if_ts_show_etb
+		subMenu = std_separator
```

Comment:
Please note the "@@if_ts_show_etb" conditional thing. I believe that for configurations which do not show ETB we will have a nasty artifact with a en empty menu.

Let me remind that changes which benefit you should not negatively affect other configurations. Prometheus would be an example which ts_show_etb false. Please invoke gen_config and test how would rusefi_prometheus.ini look